lib/wsdl.js

```diff
@@ -1957,5 +1975,13 @@ function open_wsdl(uri, options, callback) {
   return wsdl;
 }
 
+function clear_cache() {
+  WSDL_CACHE = {};
```

Comment:
I don't mind changing that, I don't think I've seen that problem before.  Is there some basis for this other than a feeling?